src/lfc_scripts.py

```diff
@@ -2,54 +2,79 @@
 import config
 from geopy.geocoders import Nominatim
 import folium
+from folium import FeatureGroup, LayerControl, Map, Marker
+import pandas as pd
+import sys
+
+site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
+site.login(config.username, config.api_key)
+geolocator = Nominatim(user_agent = "map")
 
 def main():
-    COMPETITION_NAME = "LoneStar2020"
-    site = mwclient.Site('torque.leverforchange.org/', 'GlobalView/', scheme="https")
-    site.login(config.username, config.api_key)
+    #Parse arguments
+    competition = sys.argv[1]
+    color = sys.argv[2]
+
+    #Setup Map
+    mapit = folium.Map(location=[48, -102], zoom_start=6)
+    loc = 'Corpus Christi'
```

Comment:
What's this used for exactly (I see the `.format(loc)` below but wasn't fully sure what that does)

I think a more detailed variable name might help; it also might belong as a constant.

Reply:
Deleted